README.md

```diff
@@ -4,19 +4,19 @@ Welcome to the repository for the [DDD East Midlands Blog](https://blog.dddeastm
 
 After much persuasion from the community, we have decided to move away from Medium and host our blogs using [Jekyll](https://jekyllrb.com/).
 
-We chose Jekyll as it is a framework we, the organisers, are familiar with and it allows posts to be submitted in markdown, which we hope will still keep the barrier to entry low.

```

Comment:
Personally, I prefer 'organisers' spelt with an 's' as 'organizers' is more of an American way of spelling the word.